src/views/storage/TakeSnapshot.vue

```diff
@@ -181,10 +181,9 @@ export default {
               catchMessage: this.$t('error.fetching.async.job.result')
             })
           }
-          this.closeAction()
         }).catch(error => {
           this.$notifyError(error)
-        })
+        }).finally(() => { this.closeAction() })
```

Comment:
The error is that `this.actionLoading` is not reset. Would be better to set it back to false in finally so the user can try creating the snapshot again in case of a failure

Reply:
It's caused because of both actionLoading not being set to false and not initiating form closure post completion of the task. And merely, closing the form should do, however, I've reset the actionLoading parameter as well.

Reply:
The form should remain open in case of an error so the user can try again without having to input all the fields all over again